src/lib/dev-environment/dev-environment-core.js

```diff
@@ -264,3 +270,36 @@ export async function getApplicationInformation( appId: number, envType: string
 
 	return appData;
 }
+
+export async function resolveImportPath( slug: string, fileName: string, searchReplace: string, inPlace: boolean ): Promise<SQLImportPaths> {
+	let resolvedPath = path.resolve( fileName );
```

Comment:
I had issues with path.resolve vs `~`, could you please use [this](https://github.com/Automattic/vip/blob/fcd2cd5084b51305a9d380ff9769d6211c552208/src/lib/dev-environment/dev-environment-cli.js#L201) instead

Some info - https://stackoverflow.com/a/37112508/11875271